view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -68,6 +68,17 @@ define(
                 return priceUtils.formatPrice(amount, quote.getPriceFormat());
             },
 
+            /**
+             * Get formatted message about installment value limitation
+             *
+             * NOTE: this value should be taken directly from capability object when it is fully implemented.
+             *
+             * @return {string}
+             */
+            getMinimumOrderText: function () {
+                return $.mage.__('Minimum order value is %amount').replace('%amount', this.getFormattedAmount(3000));
```

Comment:
This minimum amount of 3000 is now in more than one place in this JS file. Should be a constant

Reply:
I second this.

Reply:
ok.. done